.github/ISSUE_TEMPLATE/issue--feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 'Issue: Feature request'
+about: Suggest an idea/extension for Hazel
+title: Add a relevant feature title here
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Ex. -> e.g.

Reply:
Fixed typo in https://github.com/TheCherno/Hazel/pull/80/commits/bda4f8f9e22f8887fb4b046c185349f9863368a4.